src/CountUp.js

```diff
@@ -200,4 +195,95 @@ class CountUp extends Component {
   }
 }
 
+const NO_ELEMENT = -1;
+
+const decorator = (f, action) => (...args) => {
+  const result = f(...args);
+  action(result);
+  return result;
+};
+
+const isFunction = f => typeof f === 'function';
```

Comment:
Let's either use this helper in the entire code base (and move it to helpers.js or something) or skip just it.